app/src/main/java/org/oppia/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -49,11 +44,11 @@ class RevisionCardFragmentPresenter @Inject constructor(
     return viewModelProvider.getForFragment(fragment, RevisionCardViewModel::class.java)
   }
 
-  private fun logRevisionCardEvent(topicId: String, subTopicId: String) {
+  private fun logRevisionCardEvent(topicId: String, subTopicId: Int) {
     analyticsController.logTransitionEvent(
       oppiaClock.getCurrentCalendar().timeInMillis,
       EventLog.EventAction.OPEN_REVISION_CARD,
-      analyticsController.createRevisionCardContext(topicId, subTopicId)
+      analyticsController.createRevisionCardContext(topicId, subTopicId.toString())
```

Comment:
@Sarthak2601 Please check this PR description, which basically mentions that the `subtopicId` type is of `Int`, so should I update this in domain layer, app module as well proto for you. Or if this creates any issue for you, then this can be updated later too.

Reply:
I think the updation makes sense but before you do please make sure that you update the `EventBundleCreator` and its tests as well in the utility module as they have a direct link to this.

Reply:
Done. Thanks.

Reply:
PTAL at https://github.com/oppia/oppia-android/pull/1510/commits/6d54feee05383130c59940b425af2a706093d3e2

Reply:
One more change, in the `EventBundleCreatorTest.kt`, it's actually taking the string and not the int value. Please change that as well.

Reply:
Just did that. Thanks.